Docker/docker-compose.yml

```diff
@@ -10,4 +10,4 @@ services:
       HUB_PASSWORD: $HUB_PASSWORD
       SERVER_NAME: Unitystation - EU02 Staging
     volumes:
-      - /etc/UnityStation/admin:/UnityStation_Data/StreamingAssets/admin
\ No newline at end of file
+      - /etc/UnityStation/admin:/Unitystation_Data/StreamingAssets/admin
```

Comment:
`/etc/UnityStation/` – is this fine?

Reply:
This is just the file it uses on the host system, so i would have to rename that on the GER02 server, which i am too lazy to do.